### PR TITLE
[tasks/pipeline] Change usage example: remove obsolete 'pipeline.run …

### PR DESCRIPTION
**Pull request by yuri-lipnesh**
### What does this PR do?
Change usage example of tasks/pipeline

### Motivation
Command 
`% inv --help=pipeline.run`
shows output:
...
  Run a pipeline without kitchen tests on the current branch:
    inv pipeline.run --here --no-kitchen-tests
...
When trying to run it:
`% inv pipeline.run --here --no-kitchen-tests`
`No idea what '--no-kitchen-tests' is!`

Option `--no-kitchen-tests` is obsolete.
Another available option` --no-kmt-tests` does not appear in usage example.

### CODE REVIEW

**Review by amenasria — APPROVED** [review]
LGTM !

File: tasks/pipeline.py [hunk 1]
```diff
@@ -243,11 +243,11 @@ def run(
 ):
     """
     Run a pipeline on the given git ref (--git-ref <git ref>), or on the current branch if --here is given.
-    By default, this pipeline will run all builds & tests, including all kitchen tests, but is not a deploy pipeline.
+    By default, this pipeline will run all builds & tests, including all kmt tests, but is not a deploy pipeline.
```
Comment (pducolin) [inline tasks/pipeline.py:246]: **💬 suggestion**
```suggestion
    By default, this pipeline will run all builds & tests, including all kmt and e2e tests, but is not a deploy pipeline.
```

File: tasks/pipeline.py [hunk 2]
```diff
@@ -243,11 +243,11 @@ def run(
 ):
```
Comment (pducolin) [inline tasks/pipeline.py:243]: **💬 suggestion**
What if instead we use the task's help to document the command? The advantage is that if we remove a param from the task, the linter will help us removing it from the documentation too

Could not make an inline suggestion, but would be something like
```python
@task(
	help={
		"here": "Run against current branch, default False",
		"use_release_entries": "use release entries, used with deploy=True, default False",
		"deploy": "trigger deployment at the end of the pipeline, default False",
		"deploy_installer": "trigger installer deployment to staging, default False",
		"all_builds": "run builds for all architectures, default True",
		"e2e_tests": "run all e2e tests, default True",
		"kmt_tests": "tun all kmt tests, default True"
	}
)
def run(
    ctx,
    git_ref="",
    here=False,
    use_release_entries=False,
    major_versions=None,
    repo_branch="dev",
    deploy=False,
    deploy_installer=False,
    all_builds=True,
    e2e_tests=True,
    kmt_tests=True,
    rc_build=False,
    rc_k8s_deployments=False,
):
```

Comment (yuri-lipnesh) [reply tasks/pipeline.py:243]: Thank you for your comment, my only goal was to sync the example string with the actual parameters, i don't want to make any functional changes.

Comment (pducolin) [reply tasks/pipeline.py:243]: Would not be a functional change, but I understand it is a larger change than what you planned

**Review by pducolin — APPROVED** [review]

**Pull request merged**